[2024-12-05T18:38:00.704+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-05T18:38:00.715+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_recursion.get_vars manual__2024-12-05T18:37:58.750895+00:00 [queued]>
[2024-12-05T18:38:00.721+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_recursion.get_vars manual__2024-12-05T18:37:58.750895+00:00 [queued]>
[2024-12-05T18:38:00.721+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-12-05T18:38:00.731+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): get_vars> on 2024-12-05 18:37:58.750895+00:00
[2024-12-05T18:38:00.737+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=156) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-05T18:38:00.738+0000] {standard_task_runner.py:72} INFO - Started process 158 to run task
[2024-12-05T18:38:00.738+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'test_recursion', 'get_vars', 'manual__2024-12-05T18:37:58.750895+00:00', '--job-id', '390', '--raw', '--subdir', 'DAGS_FOLDER/test_recursion.py', '--cfg-path', '/tmp/tmp5d8xcb_1']
[2024-12-05T18:38:00.740+0000] {standard_task_runner.py:105} INFO - Job 390: Subtask get_vars
[2024-12-05T18:38:00.772+0000] {task_command.py:467} INFO - Running <TaskInstance: test_recursion.get_vars manual__2024-12-05T18:37:58.750895+00:00 [running]> on host c7780884450d
[2024-12-05T18:38:00.953+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='artzsh' AIRFLOW_CTX_DAG_ID='test_recursion' AIRFLOW_CTX_TASK_ID='get_vars' AIRFLOW_CTX_EXECUTION_DATE='2024-12-05T18:37:58.750895+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-05T18:37:58.750895+00:00'
[2024-12-05T18:38:00.954+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-05T18:38:00.992+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-12-05T18:38:00.999+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-05T18:38:00.999+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=test_recursion, task_id=get_vars, run_id=manual__2024-12-05T18:37:58.750895+00:00, execution_date=20241205T183758, start_date=20241205T183800, end_date=20241205T183800
[2024-12-05T18:38:01.032+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-05T18:38:01.045+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-05T18:38:01.048+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
